OneCommunityGlobal/HighestGoodNetworkApp
# Johny fix bug on create folder for tasks with subtasks

Fixe  bug ( PRIORITY MEDIUM) - 1

# Description
Creating sub-tasks in a WBS doesn’t create a new folder and makes the tasks uneditable
Logged in as Admin → Other Links → Projects → AAA USE THIS PROJECT → Click WBS Icon → Choose “Jae's WBS - USE THIS” → Choose task not in folder → Edit → Add Task → Should create folder out of parent and including new task




## Mainly changes explained:
1 - Creating sub-tasks in a WBS  create a new folder.
2 -  Make subtasks editable and can be deleted.
3 - Now coming  from the dashboard is able to see the tasks that are in the same folder, edit those tasks and delete it.

## How to test:
1 - check into current branch
2 - do `npm install` and `...` to run this PR locally
3 - log as admin user
4 - Go to dashboard→ Other Links→ Projects→ chose the Project "AAA USE THIS PROJECT"→ click  "[ Jae's WBS - USE THIS]"
→ Add a task → Create a subtask for this task
![Captura de Tela (71)](https://user-images.githubusercontent.com/39320057/209016299-06a63265-d1de-42fa-8f87-a9e3f8695a22.png)
![Captura de Tela (72)](https://user-images.githubusercontent.com/39320057/209016355-1f1c6ec6-524f-40a7-853f-80c32c4a04dc.png)

 


## Note:
If you look at the bug list medium priority 1 you will be able to see a video where Jae is explaining about the  bug and 
 there is a video on my dropbox folder explaining about this PR please watch boths before you test it. See the links of the videos below:
Jae explaining about the bug:
 [Video - 1](https://www.dropbox.com/s/xm3rs215fl9a9uh/Sub-tasks%20Folder%20Bug.mov?dl=0)
My video explaining how to test the PR :
[Video - 2](https://www.dropbox.com/s/m8pweyyuvzwb4fq/how_to_test_the_pull_request.mp4?dl=0)
